yamlToReadmeConverter.js

```diff
@@ -0,0 +1,118 @@
+const fs = require('fs');
+const path = require('path');
+const YAML = require('yamljs');
+
+const folderPaths = [];
+const filePaths = [];
+const ignoredFolders = ['.git', 'node_modules', '.travis', 'DS_Store'];
+
+const searchDirectories = (filePath = './') => {
+    fs.readdirSync(filePath).map(folder => {
+        const folderPath = path.join(filePath, folder)
+        if (fs.statSync(folderPath).isDirectory() && ignoredFolders.indexOf(folder) === -1) {
+            folderWalk(folderPath);
+        }
+    })
+};
+
+const folderWalk = (folderPath) => {
+    fs.readdirSync(folderPath).map(subfolder => {
+        if (ignoredFolders.indexOf(subfolder) === -1) {
+            const addFolderPath = path.join(folderPath, subfolder);
+            addFileAndFolderPaths(addFolderPath);
+        }
+    })
+};
+
+const addFileAndFolderPaths = (folderPath) => {
+    if (fs.statSync(folderPath).isDirectory() && fs.existsSync(path.join(folderPath, 'info.yaml'))) {
+        const addFilePath = path.join(folderPath, 'info.yaml');
+        folderPaths.push(folderPath);
+        filePaths.push(addFilePath);
+    }
+    else if (fs.statSync(folderPath).isDirectory()) {
+        folderWalk(folderPath);
+    }
+    else {
+        console.error(`Yaml file does not exist in folder: ${showPath(folderPath, '/')}!`);
+        process.exit(1);
+    }
+};
+
+const showPath = (stringPath, seperator) => {
+    let array = stringPath.split('/');
+    let start = 0;
+    let end = array.length - 2;
+    end++;
+    return array.slice(start, end).join(seperator);
+};
+
+const parseTags = (tags) => {
+    let parsedTags;
+    tags.forEach((tag, index) => {
+        if (index === 0) {
+            parsedTags = `\r\n  - ${tag}`;
+        }
+        else {
+            parsedTags = parsedTags + `\r\n  - ${tag}`;
+        }
+    });
+    return parsedTags;
+};
+
+const createReadme = (filePath, index) => {
+    YAML.load(filePath, (result) => {
+        const mapObj = {
+            '{{title}}': result.title,
```

Comment:
Title and description are not distinguished at all in the readme, which looks weird in general and especially for demos where they are identical, e.g. for the caption styling [README.md](https://github.com/bitmovin/demos/blob/ed16510027966a4f1b1f48dd9805aa3b39a543f7/player/caption-styling/README.md). The same applies to the long description